test/evergreen.yml

```diff
@@ -6080,6 +6080,7 @@ buildvariants:
     - name: make-check-test
     - name: unit-test
     - name: fops
+    - name: format-failure-configs-test
```

Comment:
Don't feel it's worth running the same task on 2 x ARM builders. Having it running on AL2 ARM should be good enough.

Reply:
As discussed, I will add a note in [WT-10664](https://jira.mongodb.org/browse/WT-10664) to revisit this decision. We are keeping the task in both variants for now.